doc/user_guide/io.rst

```diff
@@ -904,6 +906,127 @@ good approximation. Further release of HyperSpy will read the time axis more
 precisely by supporting non-linear axis.
 
 
+. _usid-format:
+
+USID
+----
+
+Background
+^^^^^^^^^^
+`Universal Spectroscopy and Imaging Data <https://pycroscopy.github.io/USID/about.html>`_
+(USID) is an open, community-driven, self-describing, and standardized schema for
+representing imaging and spectroscopy data of any size, dimensionality, precision,
+instrument of origin, or modality. USID data is typically stored in
+Hierarchical Data Format Files (HDF5) and the combination of USID within HDF5 files is
+referred to as h5USID.
+
+`pyUSID <https://pycroscopy.github.io/pyUSID/about.html>`_
+provides a convenient interface to I/O operations on such h5USID files. USID (via pyUSID)
+forms the foundation for other materials microscopy scientific python package called
+`pycroscopy <https://pycroscopy.github.io/pycroscopy/about.html>`_.
+
+If you have any questions regarding this module, please consider
+`contacting <https://pycroscopy.github.io/pyUSID/contact.html>`_  the developers of pyUSID
+
+Requirements
+^^^^^^^^^^^^
+1. Reading and writing h5USID files require the
+   `installation of pyUSID <https://pycroscopy.github.io/pyUSID/install.html>`_.
+2. Files must use the ``.h5`` file extension in order to use this io plugin.
+   Using the ``.hdf5`` extension will default to HyperSpy's own plugin
+
+Reading
+^^^^^^^
+h5USID files can contain multiple USID datasets within the same file.
+HyperSpy supports reading in one or more USID datasets
+
+Reading the sole dataset within a h5USID file:
+
+.. code-block:: python
+
+    >>> hs.load("sample.h5")
+    <Signal2D, title: HAADF, dimensions: (|128, 128)>
+
+If multiple datasets are present within the h5USID file and you try the same command again,
+**all** available datasets will be loaded.
+
+.. note::
+
+    Given that HDF5 files can accommodate very large datasets, setting ``lazy=True``
+    is strongly recommended if the contents of the HDF5 file are not known apriori.
+    This prevents issues with regard to loading datasets far larger than memory.
+
+    Also note that setting ``lazy=True`` leaves the file handle to the HDF5 file open.
+    If it is important that the files be closed after reading, set ``lazy=False``.
+
+.. code-block:: python
+
+    >>> hs.load("sample.h5")
+    [<Signal2D, title: HAADF, dimensions: (|128, 128)>,
+    <Signal1D, title: EELS, dimensions: (|64, 64, 1024)>]
+
+We can load a specific dataset using the ``dset_path`` keyword argument. setting it to the
+absolute path of the desired dataset will cause the single dataset to be loaded.
+
+.. code-block:: python
+
+    >>> # Loading a specific dataset
+    >>> hs.load("sample.h5", dset_path='/Measurement_004/Channel_003')
```

Comment:
Add `dset_path` to the docstring of [`load`](https://github.com/hyperspy/hyperspy/blob/1b32a1bbe82dc089e1517619fe7b53128a1c1e21/hyperspy/io.py#L56)?